lib/exogenesis/passengers/vim_plug.rb

```diff
@@ -0,0 +1,18 @@
+require 'exogenesis/support/passenger'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.